webknossos-datastore/proto/VolumeTracing.proto

```diff
@@ -23,6 +23,10 @@ message VolumeTracing {
     uint24 = 3;
     uint32 = 4;
     uint64 = 8;
+    int8 = 11;
+    int16 = 12;
+    int32 = 14;
+    int64 = 18;
```

Comment:
It took me a bit to see the pattern. Maybe add a comment that the id is `byteCount + (unsigned ? 0 : 10)`.